configuration/ops_cleanmodel.py

```diff
@@ -444,6 +444,15 @@ def execute(self, context):
 
         return {'FINISHED'}
 
+    def __locked_outfits_collections(self, rig_settings):
+        locked_objects = set()
+        for coll in [x for x in rig_settings.outfits_collections if x.collection != None]:

```

Comment:
I think `x.collection != None` should be `x.collection` alone or at most `x.collection is not None`

Reply:
https://stackoverflow.com/questions/3965104/not-none-test-in-python